ecosystem/manager.py

```diff
@@ -115,5 +116,25 @@ def dev_compatibility_tests(self,
                          tox_python=tox_python,
                          dependencies=["git+https://github.com/Qiskit/qiskit-terra.git@main"])
 
+    def parse_submission_issue(body_of_issue: str) -> Repository:
```

Comment:
Let's remove it from Manger class. I do not think we will be using it directly from CLI.
So we will not be running
```shell
python manager.py parse_submission_isse
```
I think this function will be part of other command, bigger command, something like
```shell
python manager.py handle_submission
```
and it will be used in this function
```python
def parse_submission_issue(body_of_issue: str) -> Repository:
     ...

class Manager:
    def handle_submission(issue_text: str):
        ...
        repo = parse_submission_issue(issue_text)
```

Reply:
Done :)